doc/_extensions/zephyr/domain/static/css/board-catalog.css

```diff
@@ -0,0 +1,224 @@
+/**
+ * Copyright (c) 2024, The Linux Foundation.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ .hidden {
+  display: none !important;
+}
+
+.filter-form {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+  margin-bottom: 20px;
+}
+
+.filter-form input[type="text"],
+.filter-form input[type="number"],
+.filter-form select {
+  font-family: var(--system-font-family);
+  font-size: 14px;
+  border-radius: 50px;
+  padding: 10px 18px;
+  flex: 1 1 200px;
+  background-color: var(--input-background-color);
+  color: var(--body-color);
+  transition: all 0.3s ease;
+  -webkit-appearance: none;
+  box-shadow: none;
+}
+
+.select-container {
+  flex: 1 1 200px;
+  position: relative;
+  display: inline-block;
```

Comment:
I think `display: inline-block;` could be either removed or made `display: block;`.
Not that it will change a thing.

Reply:
very nice catch! I did some further cleaning of the css while I was there. It's probably still slightly "bloated" though, I'll probably do even more cleaning in follow-up PRs.